CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ All notable changes to this project will be documented in this file.
 - Replace all `SystemAnimator` classes with `SystemTransitionAnimator`
 [#427](https://github.com/IBAnimatable/IBAnimatable/pull/427) by [@SD10](https://github.com/sd10)
 - `PresentationDesignable` now supports `contextFrameForPresentation` which allow you to present a controller with a custom configuration over another instead of being in fullscreen. Imitates `UIModalPresentationStyle.currentContext`
+- transition Animator classes initializer parameter changed from transitionDuration -> duration
```

Comment:
One last point: can you uppercase the first character and credit yourself about this?